int64 targetReached
float32 ballPositionX
float32 ballPositionY
